frontend/src/app/providers.tsx

```diff
@@ -19,16 +17,7 @@ import {
 import { EditorType } from "@/components/Feed/types/feed.types";
 import { AccessControlParams } from "@livepeer/react";
 import { ThirdwebProvider } from "thirdweb/react";
-
-export const config = getDefaultConfig({
-  appName: "OmniPredict",
-  projectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID as string,
-  chains: [chains.testnet],
```

Comment:
I think this chain , should be added in the wagmi.config to get the data about the Lens stuff.

Reply:
wagmiconfig file has been changed. it is in src/constants/wagmi.config.ts